### PR TITLE
Nikita templates

### PR DESCRIPTION
**Pull request by vermi4elli**
Added functionality for searching books

### CODE REVIEW

**Comment by vermi4elli** [comment]
@TheBestTvarynka You can review now

File: routing/login.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ const compare = (req, res, user) => {
   const password = user.password;
   bcrypt.compare(password, hash, (err, result) => {
     if (err) {
-      console.log(err)
+      console.log(err);
```
Comment (TheBestTvarynka) [inline routing/login.js]: :smile:  doog job)

File: routing/search.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    arr += '<div class=\'test\'><img class=\'cover\' src=\'' + row.preview + '\'><p class=\'description\'>' +
+      row.author + ' - ' + row.name +
+      '</p><div class=\'price\'>100$</div></div>';
+  });
+  */
+
+  for(let i = 0; i < 7; ++i)
+  arr += '<div class=\'test\'><img class=\'cover\' src=\'https://s2982.pcdn.co/wp-content/uploads/2014/08/HP_hc_new_3-e1407533776984.jpeg\'><p class=\'description\'>Your book</p><div class=\'price\'>100$</div></div>';
+  return arr;
+}
```
Comment (TheBestTvarynka) [inline routing/search.js]: I think now you can use forEach for prodaction.

File: routing/search.js [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+const dbconfig = {
+  connectionString: process.env.DATABASE_URL
+};
+
+function parseBooks(rows) {
+  let arr = '';
+
+  /*rows.forEach(row => {
+    arr += '<div class=\'test\'><img class=\'cover\' src=\'' + row.preview + '\'><p class=\'description\'>' +
```
Comment (TheBestTvarynka) [inline routing/search.js]: build your img src by:
[img src="https://${process.env.BUCKET}.s3.us-east-2.amazonaws.com/${row.photos[0]}" class="..."]
// in output you will get url like this: https://ubiquitous-spark.s3.us-east-2.amazonaws.com/photos/js.png

File: routing/search.js [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+  // req.body
+  const where = {};
+  for (const field in req.body) {
+    if (req.body[field] !== '') {
+      where[field] = req.body[field];
+    }
+  }
+  delete where.find_books;
+  console.log(where);
+  console.log(where);
```
Comment (TheBestTvarynka) [inline routing/search.js]: you must delete one of this logs

File: routing/search.js [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+        })
+        .then(rows => {
+          list += parseBooks(rows);
+        });
+    }
+  }
+  console.log('List = ' + list);
+  res.render('views/search', { layout: 'default', message: list });
+});
+*/
```
Comment (TheBestTvarynka) [inline routing/search.js]: pls, delete this

File: routing/login.js (hunk 1 above)
Comment (vermi4elli) [reply routing/login.js]: Actually, I don't remember myself doing this) I think it happened while merging

**Pull request merged**